packages/contracts-bedrock/src/periphery/faucet/Faucet.sol

```diff
@@ -123,7 +123,8 @@ contract Faucet {
         nonces[_auth.id][_params.nonce] = true;
 
         // Execute transfer of ETH to the recipient account.
-        SafeCall.call(_params.recipient, _params.gasLimit, config.amount, _params.data);
+        bool success = SafeCall.call(_params.recipient, _params.gasLimit, config.amount, _params.data);
```

Comment:
having a gas limit here was a mistake and adds complexity to the point of the next evm version (eof) removing the ability to pass a gaslimit to a subcall. i recommend just passing thru all gas to the subcall as a gas limit that ends up being too small will just brick the ability of the drip to work

Reply:
@tynes makes sense, I modified the call to be:

`bool success = SafeCall.call(_params.recipient, 0, config.amount, _params.data);`

I believe when gas limit is set to `0`, the call will use all remaining gas available. Let me know if that's right, thanks